extensions/wikia/TopLists/TopListHelper.class.php

```diff
@@ -682,6 +693,17 @@ public static function addItem() {
 		$result = array( 'result' => false );
 		$errors = array();
 
+		try {
```

Comment:
could you please move this logic to the separate function and call it in two places to avoid copy/pasting ?